correct default expectation for existing mrow test

bkardell opened the pull request:
Attempting to match defaults defined in https://github.com/whatwg/html/pull/5248
definitely needs scrutiny..

bkardell committed:
Update tabindex-001.html

Attempting to match defaults defined in https://github.com/whatwg/html/pull/5248
definitely needs scrutiny..

fred-wang reviewed (commented):
I guess it would be better to do it in two PR: (1) testing more linkable elements and then (2) change tabindex. Otherwise it's not easy to figure out what's happening when checking the log history.

bkardell committed:
Update tabindex-001.html

The default `.tabIndex` of an mrow should be 0 regardless of when it has an href or not, as a linkable element and matching historical (oddity) of other linkable things in HTML/SVG
